lib/order.rb

```diff
@@ -0,0 +1,38 @@
+
+class Order
+  attr_reader :basket
+
+  def initialize
+    @basket = []
+  end
+
+  def add_dish(dish, price)
+    @basket << { dish => price }
+  end
+
+  def total
+    order_total = 0
+
+    @basket.each do |item|
+      order_total += item.values[0]
+    end
+
+    order_total
+  end
+
+  def show_basket
```

Comment:
This show_basket method seems a little long, it’s also doing quite a few things. Could you take out some of this behaviour and extract it into new methods?